hw2/03-count.js

```diff
@@ -1,3 +1,34 @@
-// Add your code here
+const input = document.querySelector('input');
 
 input.addEventListener('keydown', handleKeyDown);
+
+function handleKeyDown(event) {
+  if (event.key == 'Enter') {
```

Comment:
The airbnb style guide suggests the use of "===" rather than "==" for logical equality. I think === checks value and type while == only checks value and can yield un expected results. [Heres](https://github.com/airbnb/javascript#comparison-operators--equality) a link to the guide where is suggests ===

Reply:
Awesome, thanks for catching this!